web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -220,11 +222,6 @@
       await handleGetAllAlbums();
     }
 
-    // Import hack :( see https://github.com/vadimkorr/svelte-carousel/issues/27#issuecomment-851022295
-    // TODO: Move to regular import once the package correctly supports ESM.
-    const module = await import('copy-image-clipboard');
-    canCopyImagesToClipboard = module.canCopyImagesToClipboard();
-
```

Comment:
Has this been resolved/does the library support ESM now, or was this workaround just never necessary?

Reply:
Well, this library does have ESM and CJS style exports. Doesn't appear to have changed since the addition via #1776 so not entire sure what was going on with that. The comment is also talking about an unrelated git repo, so not really sure I totally understand what was going on there. Regardless, a normal import of this dependency works perfectly.